instrumentation/elasticsearch/elasticsearch-rest-common/library/src/main/java/io/opentelemetry/javaagent/instrumentation/elasticsearch/rest/ElasticsearchRestClientTracer.java

```diff
@@ -54,6 +54,6 @@ protected String dbOperation(Void connection, String operation, String ignored)
 
   @Override
   protected String getInstrumentationName() {
-    return "io.opentelemetry.javaagent.elasticsearch-rest-common";
+    return "io.opentelemetry.elasticsearch-rest-common";
```

Comment:
We have a few of those "common" tracers -- I'm hoping that once we migrate to the Instrumenter API everywhere those common modules will just contain shared extractor code, and each instrumentation module will construct their instrumenter separately and give it a separate name that'll contain a proper version.